include/inviwo/core/datastructures/histogram.h

```diff
@@ -34,10 +34,15 @@
 
 #include <iterator>
 #include <vector>
+#include <bitset>
 
 namespace inviwo {
 enum class HistogramMode { Off, All, P99, P95, P90, Log };
 
+using HistogramSelection = std::bitset<32>;
+constexpr HistogramSelection histgramSelectionAll{0xffffffff};
```

Comment:
This refers to the channel(s) to show?
typo: histgramSelectionAll -> histogramSelectionAll